docs/_pages/releases.md

```diff
@@ -11,6 +11,7 @@ sidebar:
 
 **What's New**
 * Added `WithParameterName` extension to ease asserting on the parameter name for a thrown `ArgumentException` - [#1466](https://github.com/fluentassertions/fluentassertions/pull/1466).
+* Added `NotCompleteWithinAsync` to `TaskCompletionSourceAssertions` - [#1474](https://github.com/fluentassertions/fluentassertions/pull/1474).
```

Comment:
Don't you want to update the documentation as well?

Reply:
done